Dashboard/app/dashboard.html

```diff
@@ -24,13 +24,13 @@
     <script>
         window.jQuery || document.write("<script src='/vendorjs/js/vendor/jquery-1.8.2.min.js'><\/script>")
     </script>
-    <script src="/vendorjs/js/vendor/handlebars-1.0.rc.1.js"></script>
-    <script src="/vendorjs/js/vendor/ember-1.0.0.pre-2-36.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/handlebars.js/1.0.rc.1/handlebars.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/ember.js/1.0.0-pre.2/ember-1.0.0-pre.2.min.js"></script>
     <script src="/vendorjs/js/vendor/ember-data-rev10.min.js"></script>
     <script src="/vendorjs/js/vendor/jquery-ui-1.8.21.custom.js"></script>
```

Comment:
This version of ember-data seems to precede even the earliest tagged version so couldn't find it on any CDN.